packages/react-components/react-nav-preview/src/components/NavCategory/NavCategory.tsx

```diff
@@ -1,22 +1,19 @@
 import * as React from 'react';
-import type { ForwardRefComponent } from '@fluentui/react-utilities';
-// import { useCustomStyleHook_unstable } from '@fluentui/react-shared-contexts';
 import { useNavCategory_unstable } from './useNavCategory';
 import { renderNavCategory_unstable } from './renderNavCategory';
-import { useNavCategoryStyles_unstable } from './useNavCategoryStyles.styles';
+import { useNavCategoryContextValues_unstable } from '../useNavCategoryContextValues_unstable';
+
+import type { ForwardRefComponent } from '@fluentui/react-utilities';
 import type { NavCategoryProps } from './NavCategory.types';
 
 /**
  * NavCategory component - TODO: add more docs
  */
 export const NavCategory: ForwardRefComponent<NavCategoryProps> = React.forwardRef((props, ref) => {
   const state = useNavCategory_unstable(props, ref);
+  const contextValues = useNavCategoryContextValues_unstable(state);
 
-  useNavCategoryStyles_unstable(state);
-  // TODO update types in packages/react-components/react-shared-contexts/src/CustomStyleHooksContext/CustomStyleHooksContext.ts
```

Comment:
Did you mean to remove this todo?  If so, should the code be updated to call the custom style hook?

Reply:
added back